src/app/lib/views/player/player.js

```diff
@@ -482,6 +469,33 @@
             });
         },
 
+        onClick: function (e) {
+            if (!this.player.click) {
+                this.player.click = {};
+                this.player.click.count = 0;
+            }
+
+            this.player.click.count++;
+
+            if (this.player.click.count === 1) { // it's the first time we click
+                setTimeout(function () { // wait for double click
+                    if (!this.player.click.dblclick) { // if we didn't catch dbclick
+                        $('.vjs-play-control').click();
+                    }
+                    this.player.click.count = 0;
+                    this.player.click.dblclick = false;
+                }.bind(this), 300);
+            } else { // we already clicked, reset
+                this.player.click.count = 0;
+                this.player.click.dblclick = false;
+            }
+        },
```

Comment:
this will work, but it's not very elegant, as i suggested on slack we may want to have a common handler for 'click' and 'double click' events that is `_.throttle`d to get the edge one. what do you think ?